examples/platform/linux/Options.h

```diff
@@ -74,6 +74,7 @@ struct LinuxDeviceOptions
     const char * command                = nullptr;
     const char * PICS                   = nullptr;
     const char * KVS                    = nullptr;
+    const char * app_id                 = "";
```

Comment:
We still seem to have hard-coded paths for each example applications. Let's do a change where we "pass in the path to the pipes". Applications can have defaults if needed and those defaults could be even be PID based for, however our tests should explicitly pass in the FIFO path to BOTH applications and test scripts.